packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -128,7 +128,7 @@ describe('getPlatformInternal', () => {
       expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
         prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
-        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again. If you're running Prisma on Docker, you may also try to replace your base image with \`node:lts-slim\`, which already ships with OpenSSL installed.
+        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again.
```

Comment:
```suggestion
        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again. If you're running Prisma on Docker, switch to an image that has OpenSSL installed.
```
Maybe?

Reply:
Alright, I could also say "... that has OpenSSL installed, like `node:lts-bullseye-slim`"

Reply:
I fear that will lead people to try the `bookworm` one though (as they always want to be on the newest) and then complain again :D Let's see.

I generally still like the idea better to not have a concrete name in the error message, but instead link to a docs page that can more easily be updated and provide some explanation.

Reply:
Ok, final proposal:

- Add `If you're running Prisma on Docker, add this command to your Dockerfile, or switch to an image that already has OpenSSL installed.` to the message
- Revisit that old idea we had of adding a small Docker + Prisma tutorial to Prisma docs, and once we have that, add a pris.ly link in this warning